src/main/java/frc/robot/Robot.java

```diff
@@ -23,6 +24,7 @@
     public PowerDistribution pdp = new PowerDistribution();
     private RobotContainer m_robotContainer;
     private Command m_autonomousCommand;
+    public static final PneumaticsControlModule pneumaticsBase = new PneumaticsControlModule(0);
```

Comment:
1. The name now doesn't match, use either `PneumaticsControlModule` or `pcm`.
2. The default ID is `0` so you can use the simpler constructor: 
```suggestion
    public static final PneumaticsControlModule pneumaticsBase = new PneumaticsControlModule();
```